ArduCopter/events.cpp

```diff
@@ -54,9 +67,19 @@ void Copter::failsafe_battery_event(void)
         if (should_disarm_on_failsafe()) {
             init_disarm_motors();
         } else {
-            if (g.failsafe_battery_enabled == FS_BATT_RTL || control_mode == AUTO) {
+            if (control_mode == LAND && (g2.fs_batt_options & ParametersG2::FS_BATT_OPT_LAND_CONTINUE)) {
+                // continue landing because FS_BATT_OPTION is set to continue when in land mode
+                gcs().send_text(MAV_SEVERITY_WARNING, "BATT FS - CONTINUING LAND MODE");
+                
+            } else if (!AP_Notify::flags.autopilot_mode && (g2.fs_batt_options & ParametersG2::FS_BATT_OPT_PILOT_CONTROL_CONTINUE)) {
```

Comment:
If I understood this correctly `AP_Notify::flags` should never be used outside of that module.